packages/sdk/src/swap/sdk.ts

```diff
@@ -307,50 +300,6 @@ export class SwapSDK {
     return ApiService.getStatusV2(this.options.backendUrl, swapStatusRequest, options);
   }
 
-  /** @deprecated DEPRECATED(1.8) use encodeVaultSwapData to get the unsigned transaction data instead */
-  async executeSwap(
```

Comment:
need to update the `CHANGELOG.md`

Reply:
Have a look at the updated changelog please vault swap guru